src/main/java/com/uid2/shared/store/FileName.java

```diff
@@ -0,0 +1,24 @@
+package com.uid2.shared.store;
+
+public class FileName {
```

Comment:
is this intentionally unused? doesn't seem to be used in admin

Reply:
It is used a lot, e.g. ScopedStoreWriter

Reply:
Ah you mean that the file is in Shared and all usages are in Admin, good point. I think it will probably end up used in Shared before the end of this story. Happy to move it to Admin for now though.

Reply:
ScopedStoreWriter.java isn't included in this PR. Happy for it to be in Admin/Shared but if it's used by Admin for now then maybe Admin is best (saved you updating shared lib unnecessarily)

Reply:
Ok, will move to Admin